src/chrome/local-runtime.ts

```diff
@@ -197,6 +205,36 @@ export default class LocalRuntime {
     }
   }
 
+  private async history(): Promise<NavigationHistory> {
+    const {Page} = this.client
+    await Promise.all([Page.enable()])
```

Comment:
Is there a better place to enable `Page`? Seems like we'd want to do that at constructor time